setup.cfg

```diff
@@ -1,6 +1,6 @@
 [metadata]
 name = geoengine
-version = 0.1.0
+version = 0.2.0
```

Comment:
0.1.1 ?

Reply:
its no patch since it breaks code. I guess in versions prior to 1 it is okay to have sub-numbers :shrug: